gradle.properties

```diff
@@ -5,20 +5,20 @@ org.gradle.parallel=true
 java_version=21
 # Fabric Properties
 # check these on https://fabricmc.net/develop
-minecraft_version=1.21
-yarn_mappings=1.21+build.7
-loader_version=0.15.11
+minecraft_version=1.21.3
+yarn_mappings=1.21.3+build.2
+loader_version=0.16.9
 
 # Mod Properties
 mod_id=bluemapsignmarkers
 mod_name=BlueMap Sign Markers
-mod_version=1.21-0.4.0
+mod_version=1.21.3-0.5.0
 mod_description=A plugin for BlueMap that creates markers from signs
 maven_group=com.tpwalke2.bluemapsignmarkers
 archives_base_name=bluemapsignmarkers
 
 # Dependencies
-fabric_version=0.100.4+1.21
-fabric_loom_version=1.7-SNAPSHOT
-blue_map_version=5.3
+fabric_version=0.107.3+1.21.3
+fabric_loom_version=1.8-SNAPSHOT
+blue_map_version=5.4
```

Comment:
Do you know if BlueMap have updated to 1.21.3?

Reply:
they did! im using it on my server :)